packit_service/worker/checker/distgit.py

```diff
@@ -199,3 +199,26 @@ def pre_check(self) -> bool:
             )
 
         return valid
+
+
+class IsUpstreamTagMatchingConfig(Checker, GetSyncReleaseTagMixin):
+    def pre_check(self) -> bool:
+        if upstream_tag_include := self.job_config.upstream_tag_include:
+            matching_include_regex = re.match(upstream_tag_include, self.tag)
+            if not matching_include_regex:
+                logger.info(
+                    f"Tag {self.tag} doesn't match the upstream_tag_include {upstream_tag_include} "
+                    f"from the config. Skipping the syncing."
+                )
+                return False
+
+        if upstream_tag_exclude := self.job_config.upstream_tag_exclude:
+            matching_exclude_regex = re.match(upstream_tag_exclude, self.tag)
+            if matching_exclude_regex:
+                logger.info(
+                    f"Tag {self.tag} matches the upstream_tag_exclude {upstream_tag_exclude} "
+                    f"from the config. Skipping the syncing."
+                )
+                return False
```

Comment:
I wish we could somewhat propagate this info to dashboard to ease debugging. Hopefully the `.info` logs will be visible in logs of a job in dashboard.

Reply:
unfortunately, as the actual job will not be run, the pre-check logs will not be visible